Remove unnecessary version constraint in order to build
this with our new setuptools_scm.

--- lazy-object-proxy-1.6.0/setup.cfg
+++ lazy-object-proxy-1.6.0/setup.cfg
@@ -1,6 +1,6 @@
 [options]
 setup_requires = 
-	setuptools_scm>=3.3.1,<6.0
+	setuptools_scm>=3.3.1
 
 [flake8]
 max-line-length = 140
